service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/C2UploadedEmailContentProvider.java

```diff
@@ -33,7 +39,8 @@ public C2UploadedTemplate buildC2UploadNotificationTemplate(final CaseDetails ca
         adminTemplateForC2.setCallout(buildCallout(caseData));
         adminTemplateForC2.setRespondentLastName(getFirstRespondentLastName(caseData.getRespondents1()));
         adminTemplateForC2.setCaseUrl(getCaseUrl(caseDetails.getId()));
-        adminTemplateForC2.setDocumentLink(linkToAttachedDocument(latestC2));
+        adminTemplateForC2.setDocumentUrl(concatUrlAndMostRecentUploadedDocumentPath(
+            latestC2.getBinaryUrl(), xuiBaseUrl));
```

Comment:
It might be only me but I would prefer other order of the parameters - outcome of this method xuiBaseUrl + latestC2.getBinary... However the order passed to. the method is different, minor issue, but I think it reads better other way araound.